misc/python/genapi.py

```diff
@@ -40,6 +40,11 @@
     #'mesh send',
 ]
 
+# API Version:
+VERSION = '2.0a1'
+# minimega version (set this before rendering)
+MM_VERSION = 'UNKNOWN'
```

Comment:
You get that functionality, plus the ability to generate compliant versions
of the minimega.py file that include any fixes to the binding template down
the line.

Actually, I probably also should have bumped the API version to 2.0a2 when
we made the last change to it.
On Jun 5, 2015 12:23 PM, "David Fritz" <notifications@github.com> wrote:

> In misc/python/genapi.py
> <https://github.com/sandia-minimega/minimega/pull/230#discussion_r31827535>
> :
>
> > @@ -40,6 +40,11 @@
> >      #'mesh send',
> >  ]
> >
> > +# API Version:
> > +VERSION = '2.0a1'
> > +# minimega version (set this before rendering)
> > +MM_VERSION = 'UNKNOWN'
>
> I'm confused - is the goal to have the generated binding warn/nag if
> there's a version mismatch? If so, wouldn't you want per commit matching,
> since we currently do have breaking changes pretty much weekly?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sandia-minimega/minimega/pull/230/files#r31827535>.
>